1Password token generation manual updated (#2045)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

To generate a 1Password API token follow these steps:
1. [Sign in](https://start.1password.com/signin) to your 1Password account and click [Integrations](https://my.1password.com/integrations/active) in the sidebar.

> 2. Switch to the **Directory** tab (or use [direct link to Directory tab](https://sumologictestingapi.1password.com/integrations/directory))

Reviewer: ```suggestion
2. Switch to the **Directory** tab (or use [direct link to Directory tab](https://sumologictestingapi.1password.com/integrations/directory)).
```

> 3. Choose the **Sumo Logic** integration in **Events Reporting** section. 

Reviewer: ```suggestion
3. Go to the **Events Reporting** section and choose the **Sumo Logic** integration. 
```

> 4. Enter System Name of your convenience (default value Sumo Logic would be fine in most cases) and click **Add Integration**.

Reviewer: ```suggestion
4. Enter System Name of your convenience (default value `Sumo Logic` would be fine in most cases) and click **Add Integration**.
```

> 5. On the next page enter Token Name. This can be any string which will help you to recognize tokens generated for different environments like Production, Stage, Dev. 
> 6. In section **Events to Report** leave enabled event sources which you want to share using the token. Then click **Issue Token**.
> 7. On last page click icon on the right to the token string to copy it to the clipboard. You can also select **Save in 1Password** to store it for further reference. 
> 8. Then click **View Integration Details** to see summary and **Learn More** button pointing to 1Password App installation manual.

Reviewer: ```suggestion
5. On the next page, enter a Token Name. This can be any string which will help you to recognize tokens generated for different environments like Production, Stage, Dev. 
6. In the section **Events to Report**, leave enabled event sources which you want to share using the token. Then click **Issue Token**.
7. On the last page, click icon on the right to the token string to copy it to the clipboard. You can also select **Save in 1Password** to store it for further reference. 
8. Click **View Integration Details** to see summary and click **Learn More** pointing to 1Password App installation manual.
```
